test/controllers/NinoControllerSpec.scala

```diff
@@ -40,12 +40,14 @@ import scala.concurrent.{ExecutionContext, Future}
 
 class NinoControllerSpec extends ControllerSpecBase {
 
+  // TODO: This should fail!
+  val personName = "Mark"
```

Comment:
why this val is added?

Reply:
Instead of passing the name and the message key to the view and have it be interpolated there, we're doing that at the earliest point that we have those two bits of information, which is the controller. This is just a hangover from part of that refactor